Bottom: cda5866557caa4ff0b1c3aeb31da46bd11070aeb
Top:    e26160a4d7aa7ebd8e7e0183cf0a8185df7cef15
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-02 21:21:51 -0500

Refresh of zarlink

---

diff --git a/p.sh b/p.sh
index 048957b..fe29ef0 100755
--- a/p.sh
+++ b/p.sh
@@ -3,6 +3,6 @@
 stg refresh
 git branch -D ravi1
 stg publish ravi1
-stg puglish ravi
+stg publish ravi
 git push github --all --force
